Taskfile.dist.yml

```diff
@@ -39,10 +39,18 @@ tasks:
 
   patch:
     cmds:
-      - sed 's:\(delete(req.Header, "Host")\)$://\1:' "$(go env GOROOT)/src/net/http/server.go" > x
+      - sed 's/\(delete(req.Header, "Host")\)$///\1/' "$(go env GOROOT)/src/net/http/server.go" > x
```

Comment:
Any reason we're changing this? If we want to use `/` as the separator character, we have to escape the other `/` characters with a `\`. That's why I was using `:`

Running this seems to produce an error for that reason too:

```
>  sed 's/\(delete(req.Header, "Host")\)$///\1/' "$(go env GOROOT)/src/net/http/server.go"
sed: 1: "s/\(delete(req.Header,  ...": bad flag in substitute command: '/'
```

Reply:
Which platform are you run this on? I'm running this in Ubuntu (WSL) and got no exceptions. We can change it back to match your environment.